components/listingsPage/results.js

```diff
@@ -26,20 +52,33 @@ const Results = ({ t }) => {
           borderRadius: "5px",
           padding: "15px 18px",
           width: "100%",
+          display: "flex",
+          flexDirection: "row",
+          flexWrap: "wrap",
+          justifyContent: "space-between",
         }}
       >
-        {t("headings.showing")} 1 -10 {t("headings.of")} 1000{" "}
-        {t("headings.firms")}
+        <span>
+          {t("headings.showing")}
+          {firstIndex} - {lastIndex}
+          {t("headings.of")} {offered && offered.length} {t("headings.firms")}
+        </span>
+        <span>Firms presented in no particular order</span>
```

Comment:
Do we have the translation for this text? If not available yet can we ensure it's on a card to be added? 🙏

Reply:
Though this has been fixed, there is no translation for any text yet on TAD consumer. I guess that is the last thing they want me to do.